Extensions/ITokenAcquisition.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Client;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
```

Comment:
using ordering should be consistent in sample.

Reply:
This has been fixed in all the files.